x-pack/plugins/serverless_search/public/types.ts

```diff
@@ -32,6 +33,7 @@ export interface ServerlessSearchPluginStartDependencies {
   cloud: CloudStart;
   console: ConsolePluginStart;
   searchPlayground: SearchPlaygroundPluginStart;
+  searchInferenceEndpoints?: SearchInferenceEndpointsPluginStart;
```

Comment:
:x:  in order to add this here you need to `searchInferenceEndpoints` to the `optionalPlugins` in the `serverless_search/kibana.jsonc`

Reply:
Added searchInferenceEndpoints as optional plugin.